### PR TITLE
Privacy Manifest

### PR DESCRIPTION
**Pull request by porter-stripe**
## Summary
- Adds a Privacy manifest

## Motivation
https://github.com/stripe/stripe-ios/issues/3409

## Testing
Testflight distribution 

## Changelog
See diff

### CODE REVIEW

File: StripePaymentSheet.podspec [hunk 1]
```diff
@@ -17,7 +17,7 @@ Pod::Spec.new do |s|
   s.swift_version = '5.0'
   s.weak_framework                 = 'SwiftUI'
   s.source_files                   = 'StripePaymentSheet/StripePaymentSheet/Source/**/*.swift'
-  s.ios.resource_bundle            = { 'StripePaymentSheetBundle' => 'StripePaymentSheet/StripePaymentSheet/Resources/**/*.{lproj,png,xcassets,json}' }
+  s.ios.resource_bundle            = { 'StripePaymentSheetBundle' => ['StripePaymentSheet/StripePaymentSheet/Resources/**/*.{lproj,png,xcassets,json}', 'StripeCore/StripeCore/PrivacyInfo.xcprivacy'] }
```
Comment (porter-stripe) [inline StripePaymentSheet.podspec:20]: Should this be `StripePaymentSheet/StripePaymentSheet/PrivacyInfo.xcprivacy`?

File: StripeFinancialConnections.podspec [hunk 2]
```diff
@@ -19,5 +19,5 @@ Pod::Spec.new do |s|
     s.dependency   'StripeCore', "#{s.version}"
     s.dependency   'StripeUICore', "#{s.version}"
     s.source_files                   = 'StripeFinancialConnections/StripeFinancialConnections/**/*.swift'
-    s.ios.resource_bundle            = { 'StripeFinancialConnectionsBundle' => 'StripeFinancialConnections/StripeFinancialConnections/Resources/**/*.{lproj,png}' }
+    s.ios.resource_bundle            = { 'StripeFinancialConnectionsBundle' => ['StripeFinancialConnections/StripeFinancialConnections/Resources/**/*.{lproj,png}', 'StripeCore/StripeCore/PrivacyInfo.xcprivacy'] }
```
Comment (porter-stripe) [inline StripeFinancialConnections.podspec:22]: Same should it be `StripeFinancialConnections/StripeFinancialConnections/PrivacyInfo.xcprivacy`?

File: StripeCore/StripeCore/PrivacyInfo.xcprivacy [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (porter-stripe) [inline StripeCore/StripeCore/PrivacyInfo.xcprivacy:1]: Should this be added to the StripeCore podspec?

**Review by porter-stripe — COMMENTED** [review]
Mainly questions around the podspecs. And do we need anything for Identity?

File: StripeCore/StripeCore/PrivacyInfo.xcprivacy (hunk 3 above)
Comment (davidme-stripe) [reply StripeCore/StripeCore/PrivacyInfo.xcprivacy:1]: Oh weird, it's open in my text editor but there was an error when saving the .rb file. Committing it now

File: StripeFinancialConnections.podspec (hunk 2 above)
Comment (davidme-stripe) [reply StripeFinancialConnections.podspec:22]: 🤦 Good catch

**Review by yuki-stripe — APPROVED** [review]
Approving on behalf of @porter

**Pull request merged**